app/src/debug/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">DEBUG.MaterialAndroidBootstrap</string>
```

Comment:
@bbbenja 
If we want app debug to be next to app release inside launcher, consider suffixing name with "DEBUG" instead of prefixing it.